common/reducers/config/networks/staticNetworks.ts

```diff
@@ -389,6 +390,29 @@ export const INITIAL_STATE: State = {
       max: 60,
       initial: 2
     }
+  },
+  ESN: {
+    id: 'ESN',
+    name: 'EthersocialNetwork',
```

Comment:
It might be good to just call this "Ethersocial", because we append "network" to a lot of networks. It would often read as "EthersocialNetwork network". If you're fine with that though, no problem.

Reply:
no problem~
there are some fork coin exists named "ETSC" (ethereum social coin)
and some people in Korea confused on it. so we use officially `EthersocialNetwork` to be clear.

Reply:
@hackmod  :sweat_smile: